test/diagnostics/diagnostics_tests.jl

```diff
@@ -47,7 +48,10 @@ bucket_domain = ClimaLand.SphericalShell(;
     nelements = (1, 10),
 )
 
-bucket_atmos, bucket_rad = ClimaLand.prescribed_analytic_forcing(FT)
+default_params_filepath =
+    joinpath(pkgdir(ClimaLand), "toml", "default_parameters.toml")
+toml_dict = LP.create_toml_dict(FT, default_params_filepath)
```

Comment:
It would be nice to have a method `LP.create_toml_dict(FT)` which uses `default_parameters.toml` by default, since the vast majority of the time we use that toml file. If the user wants to use something else, they can provide a file or list of files that overrides this default